invenio_db/alembic/dbdbc1b19cf2_create_transaction_table.py

```diff
@@ -40,8 +41,9 @@ def upgrade():
         sa.Column('issued_at', sa.DateTime(), nullable=True),
         sa.Column('id', sa.BigInteger(), nullable=False),
         sa.Column('remote_addr', sa.String(length=50), nullable=True),
-        sa.PrimaryKeyConstraint('id')
     )
+    op.execute(CreateSequence(Sequence("transaction_id_seq")))
```

Comment:
Right, I added the autoincrement=True to the id and `flask db drop` works with the alembic commands, but there is no `transaction_id_seq`. I am not sure if this is a problem though.